src/components/ui/PageContentContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { respondFrom, breakpoints } from "../../styles";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Wrapper = styled.div`
+  padding: 0 16px;
```

Comment:
I would also consider creating global spacing definition and referring to it here. Your spacing value will probably be 8px

Reply:
this way you will have a consistent basic value for defining paddings, margins, etc

Reply:
I add new variables for spacing